src/styles.scss

```diff
@@ -329,6 +329,16 @@ body {
   .view-container {
     background-color: $white;
     padding: 1rem;
+    mat-radio-group {
+      margin-left: 10px;

```

Comment:
This adds a lot of space between the label and the choices.  Let's just make the changes to `mat-radio-button`